### PR TITLE
Fix #906: update style for new comment in existing issue

### LINKED ISSUE #906: [ui] upload image in comment
We have this 
capture d ecran 2016-01-26 a 09 51 04

But the inital design of @calexity is better IMHO.

https://github.com/webcompat/design/blob/master/2_Current%20Designs/Issues/Sketch%20and%20PSDs/IssuesStyle.psd

Any thought? @karlcow @hallvors @miketaylr @Raphseck

### PR DESCRIPTION
**Pull request by cch5ng**
@magsout had difficulty testing in IE10 (tried remote desktop) but Edge looked ok
let me know if I should try further testing or ?r

### CODE REVIEW

File: webcompat/static/css/development/components/comment.css [hunk 1]
```diff
@@ -84,6 +84,15 @@
     background : #fff;
   }
 
+    /* modifier new comment of existing bug */
+    .wc-Comment-wrapper-new {
+      display: block;
+      border-bottom: 1px dashed var(--Comment-color);
+      border-bottom-left-radius: 0;
+      border-bottom-right-radius: 0;
+      padding: 5px; /* fix placeholder text spacing */
```
Comment (magsout) [inline webcompat/static/css/development/components/comment.css]: `padding: 1em`  is better. Equal to padding in classic comment.

File: webcompat/static/css/development/components/comment.css [hunk 2]
```diff
@@ -84,6 +84,15 @@
     background : #fff;
   }
 
+    /* modifier new comment of existing bug */
+    .wc-Comment-wrapper-new {
```
Comment (magsout) [inline webcompat/static/css/development/components/comment.css]: Name of modifier is incorrect.

```css
/*  parent *
.foo {}
  /* child */
  .foo-child {}
  /* modifier */
  .foo--modifier {}
```

File: webcompat/static/css/development/components/upload-form.css [hunk 3]
```diff
@@ -47,6 +60,20 @@
       text-decoration: underline;
     }
 
+      /* modifier new label of existing bug */
+      .wc-UploadForm-label-new {
```
Comment (magsout) [inline webcompat/static/css/development/components/upload-form.css]: Same as above for name of selector

File: webcompat/static/css/development/components/upload-form.css [hunk 4]
```diff
@@ -47,6 +60,20 @@
...
+        font-weight: 400;
+        text-align: left;
+        text-decoration: none;
+      }
+      
+      /* link */
+      .wc-UploadForm-label-link {
```
Comment (magsout) [inline webcompat/static/css/development/components/upload-form.css]: -> `.wc-UploadForm-label--link {} `

**Comment by magsout** [comment]
@cch5ng except some type, r+ 

Good job @cch5ng thanks :+1:

File: webcompat/static/css/development/components/comment.css (hunk 2 above)
Comment (cch5ng) [inline webcompat/static/css/development/components/comment.css]: @magsout thanks for the review. I think either way with --new or --comment the purpose of the style will be a little ambiguous just coming from the css. I feel new is slightly more descriptive but can replace it with comment.

I assume the editor will probably know the style's use by inspection from the webpage.

**Comment by magsout** [comment]
:+1:

**Pull request merged**